src/main/java/data_gateway/CalendarManager.java

```diff
@@ -10,6 +10,8 @@
 
     boolean addEvent(CalendarEventModel eventData);
 
+    boolean completeEvent(String eventName);
```

Comment:
change it to `markEventAsCompleted`?

Reply:
also, this should take in id instead of name because that is the unique identifier of events

Reply:
can we merge in tahseens getId pr cause I'll need that for this